cloud/azure/locals.tf

```
@@ -0,0 +1,4 @@
+locals {
+  sfx_suffix               = var.sfx_integration_name_suffix == "" ? "" : format("-%s", var.sfx_integration_name_suffix)
+  default_integration_name = format("AzureClaranetIntegration%s", sfx_suffix)
```

I'd rather not put `Claranet` in open modules and it would make sense to add SignalFX here:

```suggestion
  default_integration_name = format("AzureSignalFXIntegration%s", sfx_suffix)
```